tuneme/forms.py

```diff
@@ -0,0 +1,38 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+
+class LoginForm(forms.Form):
+    username = forms.CharField(widget=forms.TextInput(attrs={
+        'placeholder': _('Username'),
+    }))
+    password = forms.CharField(widget=forms.PasswordInput(attrs={
+        'placeholder': _('4 Digit PIN'),
+    }))
+
+
+class RegistrationForm(forms.Form):
+    username = forms.CharField(widget=forms.TextInput(attrs={
+        'placeholder': _('Username'),
+    }))
+    password = forms.CharField(widget=forms.PasswordInput(attrs={
+        'placeholder': _('4 Digit PIN'),
+    }))
+
+
+class EditProfileForm(forms.Form):
+    alias = forms.CharField(widget=forms.TextInput(attrs={
+        'placeholder': _('Display Name'),
+    }))
+
+
+class ProfilePasswordChangeForm(forms.Form):
```

Comment:
@miltontony they are inherited from molo.profile.forms, I'm just adding a placeholder to it. unless I'm misunderstanding what you're saying :)